src/current/releases/v1.0.md

```diff
@@ -16,9 +16,7 @@ keywords: gin, gin index, gin indexes, inverted index, inverted indexes, acceler
 
 {% include releases/testing-release-notice.md major_version=vers %}
 
-Get future release notes emailed to you:
-
-{% include_cached marketo.html %}
+{% include releases/whats-new-intro.md major_version=major_version %}
```

Comment:
The include isn't actually parameterized in the current implentation, but inherits `page.major_version` from the page that is including it. If you wanted to support explicitly passing in a major version to the include, you would refer to it within the include as `{{ include.major_version }}` instead. Same suggestion for all the others below.
```suggestion
{% include releases/whats-new-intro.md %}
```